boat-scaffold/src/main/java/com/backbase/oss/codegen/angular/BoatAngularGenerator.java

```diff
@@ -69,6 +70,7 @@ public class BoatAngularGenerator extends AbstractTypeScriptClientCodegen {
     public static final String HAS_EXAMPLES = "hasExamples";
     public static final String PATTERN = "pattern";
     protected String foundationVersion = "6.6.7";
+    protected String specVersion = "1.0.0";
```

Comment:
Since this var is being initialised to "1.0.0" will the specVersion also have a default? I think we don't wan't to give it a default so it will be omitted from the package.json if not provided in POM properties.